inc/TRestRawBaseLineCorrectionProcess.h

```diff
@@ -24,44 +24,42 @@
 #define RESTProc_TRestRawBaseLineCorrectionProcess
 
 #include "TRestEventProcess.h"
+#include "TRestRawReadoutMetadata.h"
 #include "TRestRawSignalEvent.h"
 
 class TRestRawBaseLineCorrectionProcess : public TRestEventProcess {
    private:
     // We define specific input/output event data holders
-    TRestRawSignalEvent* fInputEvent;   //!
-    TRestRawSignalEvent* fOutputEvent;  //!
+    TRestRawSignalEvent* fInputEvent;                     //!
+    TRestRawSignalEvent* fOutputEvent;                    //!
+    TRestRawReadoutMetadata* fReadoutMetadata = nullptr;  //!
 
     void Initialize() override;
 
     /// It defines the signals id range where analysis is applied
-    TVector2 fSignalsRange = TVector2(-1, -1);
+    TVector2 fSignalsRange = {-1, -1};
```

Comment:
While we use `TVector2` for storing pairs of data, I think it's better to use `std::pair` for this.

You don't need to do this in this PR if you don't want to but it's just something to keep in mind for the future.